src/main/java/graphql/ExecutionInput.java

```diff
@@ -278,38 +278,7 @@ public Builder context(Object context) {
             return this;
         }
 
-        /**
-         * The legacy context object
-         *
-         * @param contextBuilder the context builder object to use
-         *
-         * @return this builder
-         *
-         * @deprecated - the {@link ExecutionInput#getGraphQLContext()} is a fixed mutable instance now
-         */
-        @Deprecated(since = "2021-07-05")
-        public Builder context(GraphQLContext.Builder contextBuilder) {
-            this.context = contextBuilder.build();
-            return this;
-        }
```

Comment:
This removes the context builder, do you still want to keep `private final Object context;` in ExecutionInput?